README.md

```diff
@@ -3,23 +3,25 @@
 [![Contributors](https://img.shields.io/github/contributors-anon/ralim/ironos?color=blue&style=flat)](https://github.com/Ralim/IronOS/graphs/contributors)
 [![Latest Release](https://img.shields.io/github/v/release/ralim/IronOS)](https://github.com/Ralim/IronOS/releases/latest)
 
-# IronOS - Flexible Soldering iron control Firmware
+# IronOS - Open Source Flexible Firmware for Soldering Hardware
 
 _This repository was formerly known as TS100, it's the same great code. Just with more supported devices._
 
-Originally conceived as an alternative firmware for the TS100, this firmware has evolved into a complex soldering iron control firmware.
+Originally conceived as an alternative firmware for the _TS100_, this firmware has evolved into a complex soldering hardware control firmware.
 
-The firmware implements all of the standard features of a 'smart' soldering iron, with lots of little extras and tweaks.
-I highly recommend reading the installation guide fully when installing on your iron. And after install just explore the settings menu.
+The firmware implements all of the standard features of a _smart_ soldering hardware, with lots of little extras and tweaks.
+I highly recommend reading the installation guide fully when installing on your device. And after install just explore the settings menu.
 
-For soldering irons that are designed to be powered by 'smart' power sources (PD and QC), the firmware supports settings around the negotiated power and voltage.
-For soldering irons that are designed to be powered by batteries (TS100 & Pinecil), settings for a cutoff voltage for battery protection are supported.
+For soldering hardware that are designed to be powered by _smart_ power sources such as _PD_ or _QC_, the firmware supports settings around the negotiated power and voltage.
+For soldering hardware that are designed to be powered by batteries (_TS100_ & _Pinecil_), settings for a cutoff voltage for battery protection are supported.
```

Comment:
typo : “soldering irons are” vs. “soldering hardware is”

Reply:
@yo3fxy
Please create a PR to fix this. 😊